docs/src/references/bridged-topics.md

```diff
@@ -64,6 +64,17 @@ The Azure topics are prefixed by `az/`.
  Any message published by Azure on one the subtopics of `devices/{device_id}/messages/devicebound/#`
  is republished here.
  
+ ## AWS MQTT Topics
+MQTT clients on Thin Edge device must use the below topics to communicate with the AWS cloud.
+The AWS topics are prefixed by `aws/`.
+
+ * `aws/messages/`  - Use this topic to send the messages from device to cloud.
```

Comment:
Currently this is the case, however perhaps it would be good to have a discussion regarding how the topics should be mapped and if you have any suggestions they would be welcomed.

Reply:
Yes, there are many topic-related questions. All of them must be addressed in cooperation with the authors of this [other proposal for AWS support](https://github.com/thin-edge/thin-edge.io/pull/1347).

@samuel-walsh-red and @Bravo555 can you initiate a discussion around the topics used for AWS?

Reply:
As for me and my PR, I don't have much experience with AWS, and I'm not that familiar with the project, so I don't have any suggestions or opinions regarding topic mapping. Seing how this PR includes features that mine lacks, eg. a mapper and certificate signing request, and seems more complete, I think it makes sense to have it be the base for AWS support.

Reply:
My initial thoughts were to have the `aws/messages/` topic for publishing from the device to the cloud (and of course any sub topics of this for specific needs). However, I also do not have any particularly strong feelings about this and I am open to suggestions if this is not desired. @didier-wenzek  do you have any opinion on how the topic mapping should look?